### PR TITLE
Add SERVICE_NOTICE_HTML env var and display it on login page if set

### PR DESCRIPTION
**Pull request by nick-gravgaard**

### CODE REVIEW

File: environment.sh [hunk 1]
```diff
@@ -1,10 +1,12 @@
 #!/bin/sh
 export SETTINGS='dev'
-export REGISTER_TITLE_API='http://landregistry.local:8004/'
-export LOGGING_CONFIG_FILE_PATH='logging_config.json'
+
+export APPLICATION_SECRET_KEY='secretkeyshouldberandom'
```
Comment (camillebaldock) [inline environment.sh:4]: :+1: on reordering :)

**Comment by camillebaldock** [comment]
## Acceptance criteria
- It doesn't support HTML content and if I trust the acceptance criteria listed on the story it should?

## Position on page

_yup this beautiful way of doing test data is in the open for all to see, "booya" seems to be one of my favourite phrases for some reason_

Maybe that box should be underneath the page title?
Also maybe not red for error but more an orange warning-y shade?

screen shot 2015-06-17 at 11 46 39

**Comment by Demotive** [comment]
The text wrap bug isn't of huge importance - it's the browser trying to wrap text, where there's no spaces or good places to wrap.
In a "realistic" scenario text would wrap normally :)

**Comment by camillebaldock** [comment]
@Demotive completely agreed to be honest don't think the warning is ever going to contain a long uninterrupted string without spaces ;)  @nickgravgaard please ignore that, removing it from previous comment :)

**Pull request merged**